tests/git_repo_mock.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, origin, remote_repos):
 
         self.remote_repos = remote_repos
         self._local_repo = GitRepoModel()
-        self._remotes = dict(origin=origin)
+        self._remotes = {"origin": origin}
```

Comment:
Just out of curiosity, was it so that the `dict` approach is a Python 2 thingie or something?

Reply:
Not sure :thinking: I have seen VS Code complaining about it, but not until this `pylint` updated it was actually turned into a failure.